server/storage/rethinkdb.go

```diff
@@ -29,7 +35,22 @@ type RDBTUFFile struct {
 
 // TableName returns the table name for the record type
 func (r RDBTUFFile) TableName() string {
-	return "tuf_files"
+	return TUFFileTableName
+}
+
+// Change defines the the fields required for an object in the changefeed
+type Change struct {
```

Comment:
We still write using a `SQLChange` for the gorm side. The SQL databases -> Go will coerce fine on read, but there were some interesting issues on the insert side where `""` in the ID was generating an error at the database level as it was being interpreted as an explicit `NULL` rather than an empty value that should be populated with an auto-increment ID.

Reply:
It's interesting that even though `SQLChange` specifies a custom column name for `GUN` that it reads fine (since in our tests we check the GUN value).  For safety sake, since we seem to be relying on some underlying behavior of gorm or SQL database -> go, could we also add assertions that the `SHA256` value is what is expected (since that field also specifies a custom SQL column name?)

Also, would it make sense to make `SQLChange` unexpected, since it's an internal implementation and we are actually returning the `Change` struct?  (And add a comment stating why?)

Reply:
I can also just add the `sql` tags to the `Change` object

Reply:
SQLChange and Change are both concrete types and the various storage types are implementing an interface we use at a higher level so type safety means we can never accidentally return a SQLChange

Reply:
Agree that we can't accidentally return a `SQLChange`, I was just suggesting unexporting it and the comment for readability and future maintainership purposes.  

+1 on adding the sql tags to the `Change` object too.